client/dashboard/app/primary-menu/index.tsx

```diff
@@ -12,6 +12,11 @@ function PrimaryMenu() {
 			{ supports.domains && <Menu.Item to="/domains">{ __( 'Domains' ) }</Menu.Item> }
 			{ supports.emails && <Menu.Item to="/emails">{ __( 'Emails' ) }</Menu.Item> }
 			{ supports.plugins && <Menu.Item to="/plugins/manage">{ __( 'Plugins' ) }</Menu.Item> }
+			{ supports.themes && (
+				<Menu.ExternalItem href="/themes" className="dashboard-menu__item">
```

Comment:
The's a weird asymmetry between the way it works with `Menu.ItemLink` and the way it works with `Button`.

With `Menu.ItemLink`, in the mobile menu, it's the "as" component that controls the styling. You don't need to pass a custom class name because `Menu.ItemLink` knows how to style itself.

But with `Button`, in the desktop menu, the "as" component doesn't know how to style itself. It's the `Menu.ExternalItem` being styled with the custom class.

Maybe it's not a bit deal and it would be overkill. But should it look more like this?

```suggestion
				<Menu.ExternalItem as={ Menu.NonRouterItemLink } href="/themes">
```

And `Menu.NonRouterItemLink` is just a `<Button>` except with the correct styling to be a menu item link.

It might not be overkill if we ever expect there to be a second external link in the main nav.